shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/resources/ManagedBridgeStatus.java

```diff
@@ -16,25 +22,18 @@
  */
 public class ManagedBridgeStatus extends CustomResourceStatus {
 
-    public static final String SECRET_AVAILABLE = "SecretAvailable";
-    public static final String CONFIG_MAP_AVAILABLE = "ConfigMapAvailable";
-    public static final String KNATIVE_BROKER_AVAILABLE = "KNativeBrokerAvailable";
-    public static final String AUTHORISATION_POLICY_AVAILABLE = "AuthorisationPolicyAvailable";
-    public static final String NETWORK_RESOURCE_AVAILABLE = "NetworkResourceAvailable";
-
-    private static final HashSet<Condition> CONDITIONS = new HashSet<>() {
-        {
-            add(new Condition(ConditionTypeConstants.READY, ConditionStatus.Unknown));
-            add(new Condition(SECRET_AVAILABLE, ConditionStatus.Unknown));
-            add(new Condition(CONFIG_MAP_AVAILABLE, ConditionStatus.Unknown));
-            add(new Condition(KNATIVE_BROKER_AVAILABLE, ConditionStatus.Unknown));
-            add(new Condition(AUTHORISATION_POLICY_AVAILABLE, ConditionStatus.Unknown));
-            add(new Condition(NETWORK_RESOURCE_AVAILABLE, ConditionStatus.Unknown));
-        }
-    };
+    private static Set<Condition> getInitialConditions() {
+        Set<Condition> conditions = new HashSet<>();
+        conditions.add(new Condition(DP_SECRET_READY_NAME, ConditionStatus.Unknown));
+        conditions.add(new Condition(DP_CONFIG_MAP_READY_NAME, ConditionStatus.Unknown));
+        conditions.add(new Condition(DP_KNATIVE_BROKER_READY_NAME, ConditionStatus.Unknown));
+        conditions.add(new Condition(DP_AUTHORISATION_POLICY_READY_NAME, ConditionStatus.Unknown));
+        conditions.add(new Condition(DP_NETWORK_RESOURCE_READY_NAME, ConditionStatus.Unknown));
+        return conditions;
+    }
 
     public ManagedBridgeStatus() {
-        super(CONDITIONS);
+        super(getInitialConditions());
```

Comment:
`getCreationConditions()` will probably be best and an equivalent `getDeletionConditions()` for the removal.

Reply:
done